docs/tutorials/start-a-private-network/2alicebob

```diff
@@ -0,0 +1,106 @@
+---
+title: Alice and Bob Start Blockchain
```

Comment:
Is local dev network also useful in this tutorial (the network start with `--dev`)? As it's probably the first tutorial that the user interact with substrate.

Reply:
This tutorial is about starting a non-trivial multi-node network. `--dev` is useful for developing dApps and users will encounter it in tutorials about developing dApps.